#!/usr/bin/env python3

config = {
    "ip_address": "192.168.1.19",
    "G": 9.8,
    "query_rate": 0.02,

    "max_forward_throttle": 0.2,
    "max_reverse_throttle": -0.2,
    "max_steering": 1.0,
    "steering_offset": 0.0,
    "invert_steering": False,
    "pygame_display_width": 360,
    "pygame_display_height": 720,
}